docs/concepts/governance/02-process.md

```diff
@@ -3,92 +3,111 @@ id: process
+![](./images/proposal_flow.png)
 
-1. Ensure that you have at least 2.5 million UNI delegated to your address in order to submit a proposal, or find someone who has enough UNI to meet the proposal threshold to propose on your behalf.
+Phase 3 is the final step of the governance process. If this vote passes, then a change will be enacted onchain.
 
-1. Create a topic in the Proposal Discussion category on gov.uniswap.org titled &quot;Governance Proposal — [Your Title Here]&quot; and link to any relevant Snapshot polls/discussion threads as well as the code audit report. Topics that begin with &quot;Governance Proposal&quot; that have not successfully passed through the Temperature Check and Consensus Check stages will be removed by moderators.
+To create an onchain Governance Proposal:
 
-1. Call the propose() function of the Governor Bravo to deploy your proposal.
+1. Incorporate any last iterations to your proposal based on feedback prior to posting.
 
-Once the propose() function has been called, a two day voting delay will start. After voting delay is finished a seven day voting period begins. Ongoing discussion can take place in the gov.uniswap.org forum. If the proposal passes successfully, a two day timelock will follow before the proposed code is executed.
+2. Create a topic in the [Governance Forum](https://gov.uniswap.org/) titled "Governance Proposal — [Your Title Here]" and link to previous forum posts and the Temperature Check Snapshot poll.
 
-## Soft governance
+3. Create your proposal. This can be done either through an interface (e.g. [Tally](https://tally.xyz/gov/uniswap)) or through writing the calldata for more complicated proposal logic. If the proposal passed, this calldata will execute. If writing the calldata yourself, please review the logic with a qualified Uniswap community member prior to posting the proposal.
 
-The process described above lays out a structure for those wishing to host a formal vote around a particular issue.
+4. Ensure that at least 1 million UNI is delegated to your address in order to submit a proposal, or find a delegate who has enough delegated UNI to meet the proposal threshold to propose on your behalf.
 
-However, governing this system also requires a degree of &quot;meta governance&quot;, discussions that inform the direction of and the implementation processes behind policy but which don&#39;t qualify as policy themselves.
+5. Once you submit the proposal, a two-day voting delay will start. After the voting delay finishes, a ~seven-day voting period begins. If the proposal passes, a two-day timelock must pass before you can execute the proposed code.
 
-The community may discuss new ideas and strategies for governance — including changes to the three-step process outlined above — in the &quot;Governance-Meta&quot; category. On-chain voting is not necessary to make updates to off-chain processes.
+## Changes to the Governance Process
+
+Timeframe: 7 days
+
+Threshold: 40M UNI
```

Comment:
ok so this is tagged as 40m Threshold, but it's 40m quorum and 1m threshold, correct? (also all listed just above.)

Reply:
argh sorry rachel good catch, making change will push

Reply:
fixed